ui-tests/src/test/java/io/syndesis/qe/pages/customizations/connectors/ApiClientConnectors.java

```diff
@@ -0,0 +1,68 @@
+package io.syndesis.qe.pages.customizations.connectors;
+
+import static com.codeborne.selenide.Condition.exist;
+import static com.codeborne.selenide.Condition.text;
+import static com.codeborne.selenide.Condition.visible;
+import static com.codeborne.selenide.Selenide.$;
+import static com.codeborne.selenide.Selenide.$$;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriverException;
+
+import com.codeborne.selenide.SelenideElement;
+
+import io.syndesis.qe.pages.SyndesisPageObject;
+
+public class ApiClientConnectors extends SyndesisPageObject {
+
+	private static final class Element {
+		public static final By ROOT = By.cssSelector("syndesis-api-connector-list");
+		public static By CONNECTOR_TITLE = By.xpath("//div[@class='list-pf-title']");
+		public static By CONNECTORS_LIST_ITEM = By.xpath("//div[@class='list-pf-item']");
+	}
+
+	private static class Button {
+		public static By CREATE_API_CONNECTOR_RIGHT = By.xpath("//button[@title='Create API Connector']");
+	}
+
+	public void startWizard() {
+		$(Button.CREATE_API_CONNECTOR_RIGHT).shouldBe(visible).click();
+	}
+
+	@Override
+	public SelenideElement getRootElement() {
+		return $(Element.ROOT).shouldBe(exist);
+	}
+
+	@Override
+	public boolean validate() {
+		try {
+			$(Element.ROOT).should(exist);
+			return true;
+		} catch (WebDriverException wde) {
+			return false;
+		}
```

Comment:
@mastepan 
I personally still doesn't get this approach to catch `RuntimeException`. We use it everywhere else with `getRootElement().exist();`, that seems pretty reasonable to me to have that exception with all the context needed. Like what element was expected and not found. When the exception is laundered and hidden behind {true|false} assertions, there's no valuable context to debug or point at problem.

Cc @lvydra

Reply:
@dsimansk  I think that exception logging should precede returning false.

Reply:
I have taken a look on this to Selenium and Selenide. My point is that exist() doesn't wait. Therefore I use should(exist) and I think we all should. Other thing is that exist() throws ElementNotFound Exception which I haven't noticed. I'll add it.

Reply:
Well, I've deliberately referenced `getRootElement()` which contains `should()` not plain selector `$()` for this specific reason :)

Reply:
ok, ok, you're right. I missed that fact :) done